pkgs/development/python-modules/nose-randomly/default.nix

```diff
@@ -16,9 +17,10 @@ buildPythonPackage rec {
 
   checkInputs = [ numpy nose ];
 
-  checkPhase = ''
-    nosetests
-  '';
+  checkPhase = if stdenv.isDarwin then ''
+    # Work around "OSError: AF_UNIX path too long"
+    TMPDIR="/tmp" nosetests
```

Comment:
Could we do ``export TMPDIR=$(mktemp -d)`` and set that for everyone?

Also please target master. The rebuild could is not that high.

Reply:
`mktemp -d` has come up several times before, see https://github.com/NixOS/nixpkgs/pull/126642#discussion_r650556767 and https://github.com/NixOS/nixpkgs/pull/145276#discussion_r746458026.

I will change target to master.

Reply:
Updated.